tests/integration/pkg/hooks/manifests/apigateway_cr_template.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: operator.kyma-project.io/v1alpha1
 kind: APIGateway
 metadata:
-  name: test-gateway
+  name: default
```

Comment:
We have this duplicated in with the code in `api-gateway.go`. IMP it's enough to have it hardcoded in this file. We also have this `apigateway_cr_template.yaml` and a `api-gateway.yaml` with actual name templating next to it, that seems to be unused. We should clean that up.